packages/cdktf-cli/lib/get/generator/constants/provider-namespaces.ts

```diff
@@ -0,0 +1,42 @@
+export type ResourceNamespace = { name: string; comment: string };
+
+// Namespace to Resource mapping
+const aws = {
+  ACM: ["acm_certificate", "acm_certificate_validation"],
+};
```

Comment:
What if there's no mapping, e.g. when a new resource / data source is added?

Reply:
@jsteinich Suggested we could use the prefix in general, I'll resort to that 👍 

When there is no prefix I'd say we treat it as unnamespaced, e.g. if AWS adds a new service

Reply:
How do you mean "prefix by default"?

Reply:
I was thinking we could specify a prefix string rather than an explicit list of resources. That would allow new resources within a "service" to get automatically added to the namespace.

If no specified prefixes matches, it would just go into the generic aws namespace (like everything is currently). We could add more services/prefixes in the future, but those would be breaking changes (unless we added some type of pointer).

Reply:
What @jsteinich said is implemented. I also had to add a list of additional prefixes since some groups have resources without a common prefix in them (e.g. VPC)